testing/zerotier-one/APKBUILD

```diff
@@ -1,38 +1,51 @@
-# Contributor: Kyle Parisi <kyleparisi@gmail.com>
-# Maintainer: Kyle Parisi <kyleparisi@gmail.com>
+# Contributor: ZeroTier, Inc. <contact@zerotier.com>
+# Maintainer: Adam Ierymenko <adam.ierymenko@zerotier.com>
+
+## Thanks to Kyle Parisi <kyleparisi@gmail.com> for doing the original package and getting it into testing
+
+## Small changes by Leonardo Amaral <alpinelinux@leonardoamaral.com.br>
+
 pkgname=zerotier-one
-pkgver=1.1.14
+pkgver=1.2.4
 pkgrel=1
-pkgdesc="ZeroTier One allows systems to join and participate in ZeroTier virtual networks."
+pkgdesc="Join ZeroTier virtual networks from your Alpine Linux system"
 url="https://www.zerotier.com/"
 arch="all"
 license="GPL3"
-depends=""
-makedepends="nodejs linux-headers"
-install=""
+makedepends="linux-headers miniupnpc-dev"
+install="$pkgname.pre-install $pkgname.post-deinstall"
 subpackages="$pkgname-doc"
+options="!check"
 source="
-	$pkgname-$pkgver.tar.gz::https://github.com/zerotier/ZeroTierOne/archive/${pkgver}.tar.gz
 	zerotier-one.initd
+	zerotier-one.confd
+	${pkgname}-${pkgver}.tar.gz::https://github.com/zerotier/ZeroTierOne/archive/${pkgver}.tar.gz
+	miniupnpc-system.patch
+	doc-build-bash-to-sh.patch
 	"
+builddir="$srcdir/ZeroTierOne-${pkgver}"
+
+prepare() {
+	default_prepare
+}
```

Comment:
This is redundant. `default_prepare` is implicitly called when you do _not_ define custom `prepare` function.